flowmachine/flowmachine/features/location/flows.py

```diff
@@ -193,12 +226,18 @@ def __init__(self, flow):
         cols = self.flow.column_names
         self.loc_from = ",".join([c for c in cols if c.endswith("_from")])
         self.loc_to = ",".join([c for c in cols if c.endswith("_to")])
+        self.label = ",".join(c for c in cols if c.endswith("_label"))
```

Comment:
Can this use `flow.out_label_columns` instead of filtering on column name? (And the same for `self.loc_from` and `self.loc_to` I suppose, using `flow.spatial_unit.location_id_columns`, although those haven't been added in this PR)